lib/cn/kblock_builder.c

```diff
@@ -453,7 +456,7 @@ kblock_init(
     struct kvs_cparams *cp,
     struct kvs_rparams *rp,
     struct perfc_set *  pc,
-    size_t              max_size)
+    uint32_t            max_size)
```

Comment:
size_t seems right, and is bigger...  why the change?  i hate this tool, no context...